source/configuration/read.txt

```diff
@@ -54,6 +45,20 @@ You can configure the following properties to read from MongoDB:
      - | **Required.**
        | The collection name configuration.
 
+   * - ``comment``
+     - | The comment to append to the read operation. Comments appear in the 
+         output of the :manual:`Database Profiler. </reference/database-profiler>`
```

Comment:
S: since you are linking to the Database Profiler Output page, change the link text to reflect that:
** same applies to the write page option description

```suggestion
         :manual:`output of the Database Profiler. </reference/database-profiler>`
```